Modules/Module_Processing/Module_Population/xASL_im_CreateGroupAnalysisMask.m

```diff
@@ -45,9 +45,9 @@
 end
 
 
-if x.dataset.nSubjectsSessions<16
+if x.dataset.nSubjects * x.dataset.nSessions < 16

```

Comment:
do we need ( ) ? I know that * should probably take priority over <, but above you also used ( ) so why not make sure here as well

Reply:
We don't really need () anywhere -  ( * ) < 16 or (* < 16) isn't necessary. 
But inserting them from time to time is useful to avoid mistakes.

So this here can stay as it is unless you want to have it to make it easier readable. Then I can add them...